pkgs/servers/kanidm/default.nix

```diff
@@ -19,7 +19,7 @@ let
 in
 rustPlatform.buildRustPackage rec {
   pname = "kanidm";
-  version = "1.1.0-rc.16";
+  version = "1.1.0-rc.16-unstable-2024-03-02";
```

Comment:
```suggestion
  version = "1.1.0-rc.16-5-g4c88d6b27";
```

Reply:
I’m not sure what git hash that is, but this package has already moved again. https://github.com/NixOS/nixpkgs/pull/299080